src/tentaclio/protocols.py

```diff
@@ -51,13 +49,13 @@ class Writer(_Writer[Any], Protocol):
     ...
 
 
-class ByteReader(_Reader[ByteString], Protocol):
```

Comment:
typing.BytesString is [deprecated in 3.9](https://www.python.org/dev/peps/pep-0585/) which
is causing mypy issues when using io.BytesIO